src/BuildIt.Forms/Samples/BuildIt.forms.Sample.Core/ViewModels/MainViewModel.cs

```diff
@@ -114,11 +116,21 @@ public void SwitchStates()
                 await item.Init();
                 items.Add(item);
             }
+
             Items.Fill(items);
             MoreItems.Fill(items);
+            // need to request runtime permissions for using the camera
+            var results = await CrossPermissions.Current.RequestPermissionsAsync(Permission.Camera);
+            if (results.ContainsKey(Permission.Camera))
```

Comment:
I think BuildIt.General has an extension to get value, or default if it doesn't exist

Reply:
Yep though requires casting results to IDictionary and the dictionary has enum values. Letting it return the default (which I think would be saying the permission was denied) could be a bit misleading/inaccurate